### DISCUSSION TITLE
Docker and private modules error in doc

### DISCUSSION THREAD

**Discussion by emmekappa** [npm/feedback#285] [Documentation] [answered]
Hi,
i think there's an issue on npm doc at this url: https://docs.npmjs.com/docker-and-private-modules

```
ROM risingstack/alpine:3.3-v4.3.1-3.0.1
ARG NPM_TOKEN  
COPY .npmrc .npmrc  
COPY package.json package.json  
RUN npm install  
RUN rm -f .npmrc
# Add your source files
COPY . .  
CMD npm start
```

`RUN rm -f .npmrc` should be done AFTER `COPY . .` otherwise it would be added again.

**Comment by lumaxis** [comment]
@emmekappa Thanks! I agree that this is odd but wouldn't be a better fix to change `COPY . .` to something more specific anyway? For example something like `COPY ./src .` or so. Otherwise the whole separate `COPY` steps beforehand to copy single files don't make much sense either 🤔

**Comment by MylesBorins** [comment, marked as answer]
The documentation site is now open source. If this is still a problem could you open an issues there?

https://github.com/npm/documentation
